src/modal/modal-options.class.ts

```diff
@@ -34,6 +34,10 @@ export class ModalOptions {
    * Modal data
    */
   initialState?: Object;
+  /**
+   * Modal data
```

Comment:
@omfgnuts need a better idea here ;)

Reply:
Whoops, indeed, comment should be more like `Functions to intercept the closure`